Client/Frontend/Browser/BrowserViewController/BVC+Rewards.swift

```diff
@@ -24,17 +23,18 @@ struct RewardsHelper {
             }
             
             let logOutput = extraInfo.isEmpty ? data : "\(extraInfo) \(data)"
+            var logType: OSLogType = .debug
             
             switch logLevel {
-            case .logDebug: log.debug(logOutput)
             // Response and request log levels are ledger-specific.
-            case .logInfo, .logResponse, .logRequest: log.info(logOutput)
-            case .logWarning: log.warning(logOutput)
-            case .logError: log.error(logOutput)
+            case .logDebug, .logResponse, .logRequest: logType = .debug
```

Comment:
response and request logs are noisy, I moved them to debug level for now. They still can be viewed in console app